dynawo/sources/API/JOB/test/TestJobEntry.cpp

```diff
@@ -53,6 +55,13 @@ TEST(APIJOBTest, testJobEntry) {
   ASSERT_EQ(job->getSolverEntry(), solver);
   ASSERT_EQ(job->getSimulationEntry(), simulation);
   ASSERT_EQ(job->getOutputsEntry(), outputs);
+
+  boost::shared_ptr<JobEntry> job_bis = DYN::clone(job);
```

Comment:
This only test constructor by copy and not assignment (same goes for all tests)

Reply:
OK